Source/WebCore/svg/graphics/filters/SVGFilterGraph.h

```diff
@@ -77,17 +78,19 @@ class SVGFilterGraph {
 
     RefPtr<NodeType> getNamedNode(const AtomString& id) const
     {
-        if (id.isEmpty()) {
-            if (m_lastNode)
-                return m_lastNode;
+        if (!id.isEmpty()) {
+            if (auto sourceNodes = m_sourceNodes.get(id))
```

Comment:
`m_sourceNodes.get()` does not return `sourceNodes` it returns just one `node`.

Reply:
So this local variable should be named `sourceNode` or `node`.

Reply:
Opted for `sourceNode` and pushed newer build. :-)